packages/fxa-payments-server/src/lib/test-utils.tsx

```diff
@@ -170,7 +170,8 @@ function injectStripe<P extends Object>(
       <WrappedComponent
         {...{
           ...props,
-          stripe: mockStripe as ReactStripeElements.InjectedStripeProps['stripe'],
+          stripe:
+            mockStripe as ReactStripeElements.InjectedStripeProps['stripe'],
```

Comment:
Prettier changes -- a previous version of this commit had modified this file (find-and-replace).